src/Chainweb/Pact/RestAPI/Server.hs

```diff
@@ -125,45 +133,62 @@ somePactServer (SomePactServerData (db :: PactServerData_ v c logger cas))
 
 somePactServers
     :: PayloadCas cas
+    => Logger logger
     => ChainwebVersion
     -> [(ChainId, PactServerData logger cas)]
     -> SomeServer
 somePactServers v =
     mconcat . fmap (somePactServer . uncurry (somePactServerData v))
 
 
-sendHandler :: MempoolBackend ChainwebTransaction -> SubmitBatch -> Handler RequestKeys
-sendHandler mempool (SubmitBatch cmds) =
-    Handler $
+sendHandler
+    :: Logger logger
+    => logger
+    -> MempoolBackend ChainwebTransaction
+    -> SubmitBatch
+    -> Handler RequestKeys
+sendHandler logger mempool (SubmitBatch cmds) = Handler $ do
+    liftIO $ logg Info (sshow cmds)
```

Comment:
I would propose to log JSON log messages. Those can than be (optionally) handled by a telemetry backend. Which provides the advantage that ES can discover the structure and index them accordingly.

Reply:
If the type of `cads` already has `ToJSON` and `NFData` instances, it could be passed directly to the to a JSON log function. Otherwise small wrapper data type may be used. Something like

```haskell
newtype PactCmdLog a = PactCmdLog a
    deriving (Show, ToJSON, NFData)
```